bucket/oss-browser.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.9.5",
+    "description": "ossbrowser is a graphical management tool developed by Alibaba Cloud. It provides features similar to those of Windows Explorer. Using ossbrowser, you can view, upload, download, and manage objects with ease.",
```

Comment:
```suggestion
    "description": "View, upload, download, and manage files and folders with ease.",
```

Reply:
Remove "ossbrowser is" as it is implied and unneeded.